package.json

```diff
@@ -18,40 +18,20 @@
   "author": "Aaron Chambers",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
-    "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
-    "ember-ajax": "^2.0.1",
-    "ember-cli": "2.8.0",
-    "ember-cli-app-version": "^1.0.0",
-    "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars": "^1.0.3",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-jshint": "^1.0.0",
-    "ember-cli-qunit": "^2.1.0",
-    "ember-cli-release": "1.0.0-beta.2",
```

Comment:
FYI - this addon (and the corresponding `config/release.js` file) is used for publishing this addon to npm and updating GH releases. However, https://github.com/release-it/release-it is the more modern way of doing this these days and so this repo could probably do with updating to use that. So, we can keep this change for now. Just an FYI.